### DISCUSSION TITLE
How to add starter content to WordPress themes

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#102] [Topic Ideas]
This would be a tutorial on using the Starter Content feature for themes. Despite having been around [since WordPress 4.7](https://make.wordpress.org/core/2016/11/30/starter-content-for-themes-in-4-7/), it's still a relatively unused feature. For classic themes, there is direct integration with the customizer. For themes shared in the theme repository, the starter content is integrated into the theme preview system. The [WordPress Playground](https://developer.wordpress.org/playground/) will also likely be used to launch full previews in the near-_ish_ future. Now is a good time to explore what's possible with this feature.

I haven't fully researched what this looks like with block themes yet.

**Comment by marybaum** [comment]
Love it!

**Comment by mburridge** [comment]
Tangentially related, but it might be worth mentioning [Theme Unit Test](https://codex.wordpress.org/Theme_Unit_Test). I've found it useful when developing themes.

**Comment by bph** [comment]
Created an issue for it to move forward. Please continue discussion there.

- https://github.com/WordPress/developer-blog-content/issues/114

This discussion is locked.

**Comment by justintadlock** [comment]
Closing this out based on the discussion here: https://github.com/WordPress/developer-blog-content/issues/114#issuecomment-1598819713

We can open a new topic discussion if we want to revisit this down the road.
